runtime/defaults/defaults.go

```diff
@@ -2,10 +2,26 @@ package defaults
 
 import (
 	"fmt"
+	"os"
 
 	"github.com/pokt-network/pocket/runtime/configs/types"
 )
 
+func init() {
+	initDefaultRootDirectory()
+}
+
+func initDefaultRootDirectory() {
```

Comment:
Is there a reason this couldnt go in the `init()` function directly?

Reply:
nope, just a style thing